src/plugins/tagfilter/tagfilter-en.hbs

```diff
@@ -0,0 +1,1205 @@
+							&lt;p&gt;&lt;span class=&quot;label label-default&quot;&gt;iOS&lt;/span&gt; &lt;span class=&quot;label label-success&quot;&gt;Android&lt;/span&gt;&lt;/p&gt;
+							&lt;p class=&quot;mrgn-bttm-0&quot;&gt;Your mobile business partner. A simplified all-access point for businesses to find essential information, directly interact with government resources and use tools to build your business.&lt;/p&gt;
+						&lt;/div&gt;
+					&lt;/div&gt;
+					&lt;div class=&quot;col-sm-6 col-lg-4&quot; data-wb-tags=&quot;plat__ios plat__android inst__HC&quot;&gt;
+						&lt;div class=&quot;well&quot; style=&quot;height:100%&quot;&gt;
+							&lt;h3 class=&quot;mrgn-tp-0&quot;&gt;Canada COVID-19&lt;/h3&gt;
+							&lt;p&gt;&lt;span class=&quot;label label-default&quot;&gt;iOS&lt;/span&gt; &lt;span class=&quot;label label-success&quot;&gt;Android&lt;/span&gt;&lt;/p&gt;
+							&lt;p class=&quot;mrgn-bttm-0&quot;&gt;The app will support you to track your symptoms, receive the latest updates, and access trusted resources.&lt;/p&gt;
+						&lt;/div&gt;
+					&lt;/div&gt;
+					&lt;div class=&quot;col-sm-6 col-lg-4&quot; data-wb-tags=&quot;plat__ios plat__android inst__NDandCAF&quot;&gt;
+						&lt;div class=&quot;well&quot; style=&quot;height:100%&quot;&gt;
+							&lt;h3 class=&quot;mrgn-tp-0&quot;&gt;Canadian Armed Forces App&lt;/h3&gt;
+							&lt;p&gt;&lt;span class=&quot;label label-default&quot;&gt;iOS&lt;/span&gt; &lt;span class=&quot;label label-success&quot;&gt;Android&lt;/span&gt;&lt;/p&gt;
+							&lt;p class=&quot;mrgn-bttm-0&quot;&gt;A new look for 2021! Get the latest updates from CAF leadership, access CANFORGENS, browse social media feeds, view Defence imagery and discover a variety of resources such as health, pay and pension&lt;/p&gt;
+						&lt;/div&gt;
+					&lt;/div&gt;
+					&lt;div class=&quot;col-md-6 col-lg-4&quot; data-wb-tags=&quot;plat__ios plat__android inst__HC inst__PHAC&quot;&gt;
+						&lt;div class=&quot;well&quot; style=&quot;height:100%&quot;&gt;
+							&lt;h3 class=&quot;mrgn-tp-0&quot;&gt;Canadian Guidelines on Sexually Transmitted Infections&lt;/h3&gt;
+							&lt;p&gt;&lt;span class=&quot;label label-default&quot;&gt;iOS&lt;/span&gt; &lt;span class=&quot;label label-success&quot;&gt;Android&lt;/span&gt;&lt;/p&gt;
+							&lt;p class=&quot;mrgn-bttm-0&quot;&gt;If you are a health professional, you will want to download our new Canadian Guidelines on Sexually Transmitted Infections mobile app.&lt;/p&gt;
+						&lt;/div&gt;
+					&lt;/div&gt;
+					...
+				&lt;/div&gt;
+			&lt;/div&gt;
+		&lt;/div&gt;
+	&lt;/section&gt;</code></pre>
+</details>
+
+<section id="filteredFAQ" class="wb-tagfilter provisional wb-filter" data-wb-filter='{
+		"source": "#searchFilter",
+		"selector": "article",
+		"section": ".wb-tagfilter-items",
+		"uiTemplate": "#searchuiTemplate"
+	}'>
+	<div id="searchuiTemplate" class="hidden">
+		<div class="col-sm-12">
+			<h3 class="wb-fltr-info mrgn-tp-sm">_NBITEM_ results</h3>
```

Comment:
L'exemple est broken car il n'utilise pas la notation pour personnaliser l'interface du le "filtre" plugin.
Les 2 span sont requis sinon on obtient cet erreur: `Uncaught TypeError: can't access property "textContent", document.querySelector(...) is null` lancé par: https://github.com/wet-boew/wet-boew/blob/master/src/plugins/filter/filter.js#L126-L127

Aussi, ce titre est ambigu et devrais être complémenté avec du texte additionnelle indiquant sa nature et répondant à la question "x résultats de quoi?".

Le plugin tag-filter throw une erreur car il ne reconnaît pas l'input utilisé pour le filter plugin interface: `wb-tagfilter: Filter controls require an attribute 'name'.`

Suggestion à titre informatif seulement a des fins d'essaie
```suggestion
<h3 class="wb-fltr-info mrgn-tp-sm"><span data-nbitem></span> results <span data-total hidden></span></h3>
```

Idem en français

Reply:
Effectivement, j'attendais que la PR du filter UI soit fusionnée avant d'effectuer les changements nécessaires.